Content/Tiles/Permafrost/AuroraBrick.cs

```diff
@@ -25,6 +30,17 @@ public override void DrawEffects(int i, int j, SpriteBatch spriteBatch, ref Tile
 
 			drawData.colorTint = color.MultiplyRGB(Color.White * mult);
 		}
+
+		private bool StopGrappling(On.Terraria.Projectile.orig_AI_007_GrapplingHooks_CanTileBeLatchedOnTo orig, Projectile self, Tile theTile)
+		{
+			if (theTile.TileType == TileType<AuroraBrick>())
+			{
+				self.tileCollide = true;
+				return false;
+			}
+
+			return orig(self, theTile);
```

Comment:
I think it would be nice if the hook could be stopped when it collides with the barrier tentacle instead of waiting for it to go all the way to a block or max reach

Reply:
Good idea. Will implement